icepyx/core/is2ref.py

```diff
@@ -89,83 +98,15 @@ def about_product(prod):
 
 # DevGoal: use a mock of this output to test later functions, such as displaying options and widgets, etc.
 # options to get customization options for ICESat-2 data (though could be used generally)
+@deprecated(
+    version="1.4.0", reason="order_vars() is going away, use variables() instead"
+)
 def _get_custom_options(session, product, version):
```

Comment:
Is this deprecation message correct?

Reply:
I think we can just remove this altogether, it used to be the custom options for EGI, the API we're replacing but it won't work at all anyways.

Reply:
Ok, then just remove this.